examples/stable_diffusion_xl/gm/modules/diffusionmodules/openaimodel.py

```diff
@@ -154,16 +153,11 @@ def __init__(
         self.use_scale_shift_norm = use_scale_shift_norm
         self.exchange_temb_dims = exchange_temb_dims
 
-        if isinstance(kernel_size, Iterable):
```

Comment:
not suggest to delete this since it may have rectangular kernel

Reply:
Shouldn’t MindSpore match the height and width of the input when `pad_mode` is set to `same`, even when the kernel is rectangular?

Reply:
yes if it is `same` pad_mode

Reply:
It's in the `same` mode now:
https://github.com/mindspore-lab/mindone/pull/248#discussion_r1425089106